packages/config/src/options.ts

```diff
@@ -209,7 +209,7 @@ const resolvedOptions: Array<ResolvedConfigOption> = [
     defaultValue: false,
     validation: validate.isBoolean,
     isExperimental: true,
-    canUpdateDuringTestTime: true,
+    canUpdateDuringTestTime: false,
```

Comment:
This probably should be in its own commit, but I noticed we had this enabled to be updated during test time, when it should not be able to be updated during test time.

Reply:
@AtofStryker is this a breaking change from 9.x?

Reply:
@emilyrohrbough I think it's just wrong here. Should be [fine](https://github.com/cypress-io/cypress/blob/develop/packages/config/lib/options.ts#L160) in `9.x`. My guess is it wasn't caught when merging in with the `10.x-release` branch.

Reply:
Nice catch then!

Reply:
@emilyrohrbough thank you! I did want to point out though that this did cause some issues with the config flag being set through the global config function for some of the session runner cy-in-cy tests. I moved the `session.ui` tests to point to a new directory with the session tests moved into it with the experimental flag turned on in the config in [e214495](https://github.com/cypress-io/cypress/pull/21588/commits/e2144950c1c52401830c76b673f1d3e4d4b7f4e4), [1677ca0](https://github.com/cypress-io/cypress/pull/21588/commits/1677ca07460e1def6075e944c2d6cbb38de29cbf), and [fd08b57](https://github.com/cypress-io/cypress/pull/21588/commits/fd08b57c3a3e05cbb9571717f4d6c872f98d07aa)